app/services/analytics.rb

```diff
@@ -11,6 +11,7 @@ def track_event(event, attributes = {})
     }
 
     ahoy.track(event, analytics_hash.merge!(request_attributes))
+    GoogleAnalyticsJob.perform_later(event_name: event) if Figaro.env.google_analytics_key.present?
```

Comment:
I think we want to stop the execution as early as possible. Why call a job that won't get executed? Having the conditional here also better supports testing. It feels weird to me to expect that a job is called in a test when the job is not going to actually run. I think it's cleaner to test the conditional in the Analytics spec, and then unit test the job as if it always runs. This is similar to other jobs we have that run conditionally. We only call them when we need them to run.

Also, technically, we don't need the conditional because the Staccato gem helpfully uses a No-Op Tracker if a Google Analytics key is not present. This allows us to run the test suite without having to use webmock stubs or defining our own Fakes for tests. If we were to test the conditional in the job, we'd have to set a value for the Figaro key, which would use the real Staccato tracker, which would make a call to Google Analytics that we'd have to stub (or duplicate the Staccato interface in a fake class for testing).